### PR TITLE
Entri/mighty sites template

### PR DESCRIPTION
**Pull request by evelyndearaujo-goentri**

### CODE REVIEW

**Review by wdbaker-godaddy — APPROVED** [review]
Approved

File: goentri.com.fairmint.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (pawel-kow) [inline goentri.com.fairmint.json:1]: This file is in #306, so should not be here, right?

File: goentri.com.mighty-sites.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "serviceId": "mighty-sites",
+  "serviceName": "MightySites",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up domain using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "mappingValue1, mappingValue2 are the values assigned to hosts websiteHost1, websiteHost2 subsequently",
+  "sharedProviderName": true,
+  "records": [
```
Comment (pawel-kow) [inline goentri.com.mighty-sites.json:13]: A similar comment like in #306, are those values really truly dynamic in all cases?
For me it looks like a typical APEX/www configuration, which by the way may have flaws as there might be some implementation differences if you pass `@` as a value for `%websiteHost1%`

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: goentri.com.fairmint.json (hunk 1 above)
Comment (evelyndearaujo-goentri) [reply goentri.com.fairmint.json]: Yes, you are right, it's deleted. Thank you.

**Review by pawel-kow — APPROVED** [review]
Lgtm

**Pull request merged**